go/protocol/keybase1/constants.go

```diff
@@ -144,6 +144,8 @@ const (
 	StatusCode_SCTeamImplicitBadAdd        StatusCode = 2685
 	StatusCode_SCTeamImplicitBadRemove     StatusCode = 2686
 	StatusCode_SCTeamKeyMaskNotFound       StatusCode = 2697
+	StatusCode_SCTeamBanned                StatusCode = 2702
```

Comment:
is there no more room in the 2600s for this one?

Reply:
there might well be a gap in the list

Reply:
Nope, team errors span from 2601 with `TEAM_BAD_ID                   : 2601` onwards to `TEAM_INVALID_BAN              : 2703` without any gaps. Just not all errors are defined in client.

and this just made me realize I reused the error code, oops

Reply:
2638!

Reply:
(erm, not 2638*2637*2636....)